src/main/java/bio/terra/janitor/service/janitor/ModelUtils.java

```diff
@@ -0,0 +1,66 @@
+package bio.terra.janitor.service.janitor;
+
+import bio.terra.janitor.db.TrackRequest;
+import bio.terra.janitor.db.TrackedResource;
+import bio.terra.janitor.db.TrackedResourceAndLabels;
+import bio.terra.janitor.db.TrackedResourceState;
+import bio.terra.janitor.generated.model.CreateResourceRequestBody;
+import bio.terra.janitor.generated.model.ResourceState;
+import bio.terra.janitor.generated.model.TrackedResourceInfo;
+import com.google.common.base.Preconditions;
+import com.google.common.collect.BiMap;
+import com.google.common.collect.EnumBiMap;
+import com.google.common.collect.ImmutableMap;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+
+/** Helper class for converting to and from the request/response model format. */
+public class ModelUtils {
```

Comment:
One unrelated topic, we have to write a copier function for each API layer - DB layer mapping. Is there library we can use? e.g., in protobuf, there are bunch of proto copier class I have used. 
Found this:, not sure if this is helpful https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/beans/BeanUtils.html#copyProperties-java.lang.Object-java.lang.Object-
I will give it a shot in RBS..

Reply:
There's AutoConverter at Google. I don't know if there are open source stuff. I generally find that while writing the converters does take 5 minutes of boilerplate, it's not particularly difficult to create or maintain. With some simple test coverage, it's hard to mess up. Whereas magic copier classers still need the simple test coverage and have a little more of a learning curve. But that's all definitely personal opinion and it depends on context.